src/pybind/mgr/dashboard/controllers/prometheus.py

```diff
@@ -35,6 +35,7 @@ def _api_request(self, url_suffix, params=None):
     def list(self, **params):
         return self._api_request('/alerts', params)
 
+    @ReadPermission
     @RESTController.Collection('POST')
```

Comment:
I think we should change `POST` here as it's not [HTTP](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html) compliant (`POST` is for sending data to the server, not for data retrieval).

Reply:
Agree, `GET` seems more appropriate here.

Reply:
done